ONSdigital/census-rm-case-processor
# Spike sending PubSub message to AIMS for new addresses

# Motivation and Context
Fieldworkers might discover new addresses, and they want to include them, but they need to have an entry in AIMS or else RH won't be able to launch. Case Processor needs to tell AIMS about new addresses, instead of having AIMS subscribe to the relevant Rabbit topic.

# What has changed
Added pubsub to Case Processor and got it to send messages to a topic in a project, which can. be configured to be in AIMS.

# How to test?
There are integration tests. If you want to run manually, you can point at your own GCP project by fiddling with the `application.yml`. Make sure you have run `gcloud auth application-default login` and set your `GOOGLE_APPLICATION_CREDENTIALS`. Basically, it's pretty fussy about credentials, but it will work with some hacking. You'll need a PubSub topic to publish to in your project, if you're not using the emulator.

# Links
Trello: https://trello.com/c/oBRB9Vtc
